lib/src/jpegr.cpp

```diff
@@ -709,12 +709,16 @@ status_t JpegR::decodeJPEGR(jr_compressed_ptr jpegr_image_ptr, jr_uncompressed_p
                                          DECODE_STREAM)) {
       return ERROR_JPEGR_DECODE_ERROR;
     }
+    auto gainmap_components = 0;
```

Comment:
Is this variable required?

Reply:
Looks like a future commit is resolving this. The commits can be squashed and force pushed?